invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordManagement.js

```diff
@@ -32,19 +33,24 @@ export class RecordManagement extends Component {
       isPreviewSubmissionRequest,
       currentUserId,
       accessLinksSearchConfig,
+      recordOwnerUsername,
     } = this.props;
     const { error } = this.state;
     const { id: recid } = record;
-
     const handleError = (errorMessage) => {
       console.error(errorMessage);
       this.setState({ error: errorMessage });
     };
 
     return (
       <Grid columns={1} className="record-management">
-        {permissions.can_edit && !isDraft && (
+        {permissions.can_manage && (
```

Comment:
shouldn't be this `can_manage_communities`? Are we exposing the manage button to record owners/curators ?